modules/bmo-performing-a-live-update-to-the-hostfirmwaresettings-resource.adoc

```diff
@@ -26,6 +31,11 @@ $ oc patch hostfirmwaresettings <hostname> --type merge -p \ <1>
 ----
 <1> Replace `<hostname>` with the name of the host.
 <2> Replace `<name>` with the name of the setting. Replace `<value>` with the value of the setting. You can set multiple name/value pairs.
++
+[NOTE]
+====
+Get the `FirmwareSchema` resource to determine which settings the hardware supports. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the resource.
```

Comment:
suggestion: Get the `FirmwareSchema` resource to determine which settings the hardware supports and what are the allowable values. FirmwareSchema is an auto-generated resource and is not user-updatable.